docs/smart_rollups.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Smart Rollups
+author: Davis Sawali 
+---
+
+# Smart Optimistic Rollups
+
+Rollups are a permissionless scaling for the Tezos blockchain. The idea is that anyone can originate and operate one or more rollups, increasing the overall throughput of the Tezos blockchain.
```

Comment:
sounds kind of awkward 
`Rollups are a permissionless scaling for` maybe `Rollups are a permissionless scaling implementation for`

Reply:
good catch, your suggestion sounds a lot better